nx.json

```diff
@@ -0,0 +1,46 @@
+{
+	"$schema": "./node_modules/nx/schemas/nx-schema.json",
+	"installation": {
+		"version": "19.5.0",
+		"plugins": {
+			"@nx/js": "19.5.0",
+			"nx/plugins/package-json": "19.5.0"
+		}
+	},
+	"release": {
+		"groups": {
+			"images": {
+				"projects": [ "build/*" ]
+			},
+			"deploy": {
+				"projects": [ "deploy" ]
+			}
+		},
+		"projectsRelationship": "independent",
+		"version": {
+			"conventionalCommits": true
+		},
+		"changelog": {
+			"projectChangelogs": {
+				"createRelease": "github"
+			}
+		}
+	},
+	"targetDefaults": {
+		"lint": {
+			"command": "./lint.sh {projectRoot}"
+		},
+		"test": {
+			"command": "test/test.sh"
+		},
+		"build": {
+			"command": "build/build.sh {projectName}"
+		},
+		"update-commits": {
+			"command": "build/update-commits.sh {projectName}"
+		},
+		"nx-release-publish": {
+			"command": "build/build.sh {projectName}"
+		}
```

Comment:
I find it confusing that `build` and `nx-release-publish` are actually the same commands.

Reply:
Mostly agree, but what do you suggest? Could rename the script to `build-or-publish.sh`? As it is it does at least mirror what we're for now doing in each case `buildx build`.

Reply:
Ideally I'd like to have it explicit, maybe via arguments, like:

```
		"build": {
			"command": "build/build.sh {projectName}"
		},
		"nx-release-publish": {
			"command": "build/build.sh --publish {projectName}"
		}
```

Reply:
Ha! Totally will do that. It was in fact how it was in my original implementation, and I took it out as a way for insisting that publish only happened through nx managed command which is currently the trigger.